govc/test/import.bats

```diff
@@ -50,6 +50,7 @@ load test_helper
 }
 
 @test "import.ovf with name as argument" {
+  skip
```

Comment:
We don't want to always skip these.  Let's add something like the vcsim_env env helper, call it 'skip_if_vca' - the skip_if_vca function will check for an env var to see if running on vca (fine to set via drone.yml too).  And add a message, something like: skip "disabled on vCA"